drivers/entropy/entropy_cc310.c

```diff
@@ -84,7 +95,21 @@ static int entropy_cc3xx_rng_init(const struct device *dev)
 {
 	(void)dev;
 
-	#if !defined(CONFIG_SPM)
+	#if defined(CONFIG_BUILD_WITH_TFM)
+		int ret = 0;
+		enum tfm_status_e tfm_status;
+
+		tfm_status = tfm_ns_interface_init();
```

Comment:
Can you qualify that this can be runnable potentially multiple times?
If not then we may want to move this initialization to its own "driver" of sorts, specifically handling kernel-level initialization of TF-M/PSA

Adding @oyvindronningstad to this PR to give input on this

Reply:
Actually, after #3728, this call can and should be removed.

Reply:
We keep it in here, to ensure we can run this "right now".  Can we remove in PR #3728 if this goes in before?

Reply:
Of course